conversion/conversion.go

```diff
@@ -1024,9 +1025,9 @@ func GetBanner(now time.Time, db string) string {
 func ProcessDump(driver string, conv *internal.Conv, r *internal.Reader) error {
 	switch driver {
 	case MYSQLDUMP:
-		return mysql.ProcessMySQLDump(conv, r)
+		return common.ProcessDbDump(conv, r, mysql.MysqlDbDump{})
 	case PGDUMP:
-		return postgres.ProcessPgDump(conv, r)
+		return common.ProcessDbDump(conv, r, postgres.PostgresDbDump{})
 	default:
 		return fmt.Errorf("process dump for driver %s not supported", driver)
 	}
```

Comment:
I wonder if sources/common/dbdump.go is a better place for this factory method now? (similarly suggest moving ProcessInforSchema() to sources/common/infoschema.go etc)

Reply:
Agreed. Plan to take this up in further refactoring as outlined in part 3 of issue #178